src/ppx/graphics_util.cpp

```diff
@@ -31,6 +32,178 @@
 namespace ppx {
 namespace grfx_util {
 
+namespace {
+
+// Start planar image helper functions
+
+// Gets the height of a single plane, in terms of number of pixels represented.
+// This doesn't directly correlate to the number of bits / bytes for the plane's
+// height. The value returned can be used in a copy-image-to-buffer command.
+// plane: The plane to get the height for (containing information about the
+//        color components represented in the plane).
+// subsampling: The type of subsampling applied to chroma values for the image
+//              (e.g. 444, 422, 420).
+// imageHeight: The height of the image, in pixels, with no subsampling applied.
+uint32_t GetPlaneHeightForCopy(
```

Comment:
maybe better without `ForCopy` since this could be used for other purpose than copy-image-to-buffer?

Reply:
Replaced with 'InPixels', since it's the pixel height / width of a plane after subsampling. I'm not sure if there are uses besides copy where we'd use this, but just in case, this is at least more descriptive.